schemas/_common-schema.json

```diff
@@ -99,6 +99,21 @@
       },
       "readOnly": true
     },
+    "focus": {
```

Comment:
since it is not user facing, should it be in `_dd` instead?

Reply:
I revolved the discussion but in the RFC, the logic was: if it can be useful to us, it can be useful to our client. Hence it should not be private.

Reply:
Have not seen this one, ok then